license.md

```diff
@@ -1,23 +1,7 @@
-Copyright (c) 2015, SilverStripe
-All rights reserved.
+Copyright (c) 2015 Christopher Pitt
```

Comment:
Hi @assertchris, see the internal note, since some of this gets into employment law WRT you being an employee of SilverStripe Ltd. In my opinion this is definitely incorrect though (even ignoring the employment thing, rewrites are still often considered derivative works, unless they were clean room).